pkgs/faf-client/update.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq gnugrep gnused
```

Comment:
This will error out as there isn't a `NIX_PATH` set. You can either set `NIX_PATH=flake:nixpkgs=nixpkgs` which will pick it up from the registry, or use `nix shell`-style shebang, like this:
```sh
#!/usr/bin/env -S nix shell nixpkgs#curl nixpkgs#jq nixpkgs#gnugrep nixpkgs#gnused -c bash
```

Reply:
can you tell me which situations it will error out in? it works on my machine™

Reply:
Right, it also works on mine. But in a GH runner it will fail with this:
```nix
Run pkgs/osu-lazer-bin/update.sh
error: file 'nixpkgs' was not found in the Nix search path (add it using $NIX_PATH or -I)

       at «string»:1:25:

            1| {...}@args: with import <nixpkgs> args; (pkgs.runCommandCC or pkgs.runCommand) "shell" { buildInputs = [ (jq) (moreutils) ]; } ""
             |                         ^
(use '--show-trace' to show detailed location information)
```